test/fuzz/transformation_add_bit_instruction_synonym_test.cpp

```diff
@@ -991,7 +991,9 @@ TEST(TransformationAddBitInstructionSynonymTest, DISABLED_DifferentSingedness) {
 ; main function
          %37 = OpFunction %3 None %4
          %38 = OpLabel
-         %39 = OpNot %200 %5 ; bit instruction
+         %39 = OpBitwiseOr %200 %5 %6 ; bit instruction
+         %40 = OpBitwiseAnd %2 %7 %8  ; bit instruction
```

Comment:
It would make sense to use these instructions in the test somehow as well. For example, you can check that the transformation is inapplicable for the first instruction and can be applied for the second one.

Reply:
Done.